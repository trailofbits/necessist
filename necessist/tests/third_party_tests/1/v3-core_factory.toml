url = "https://github.com/Uniswap/v3-core"
test_files = ["test/UniswapV3Factory.spec.ts"]
full = true
config_mandatory = true

# smoelius: Some of the tests take close to the time limit on macOS.
[config]
ignored_tests = ["pool bytecode size", "fails if caller is not owner"]
