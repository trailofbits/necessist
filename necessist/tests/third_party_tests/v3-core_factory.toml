url = "https://github.com/Uniswap/v3-core"
test_files = ["test/UniswapV3Factory.spec.ts"]
full = true
