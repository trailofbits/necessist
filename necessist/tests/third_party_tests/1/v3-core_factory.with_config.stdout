18 candidates in 1 test file
$DIR/test/UniswapV3Factory.spec.ts: dry running
$DIR/test/UniswapV3Factory.spec.ts: mutilating
$DIR/test/UniswapV3Factory.spec.ts:106:27-106:91: `.createPool(TEST_ADDRESSES[0], TEST_ADDRESSES[0], FeeAmount.LOW)` failed
$DIR/test/UniswapV3Factory.spec.ts:118:27-118:81: `.createPool(TEST_ADDRESSES[0], TEST_ADDRESSES[1], 250)` failed
$DIR/test/UniswapV3Factory.spec.ts:122:36-122:103: `.createPool(TEST_ADDRESSES[0], TEST_ADDRESSES[1], FeeAmount.MEDIUM)` failed
$DIR/test/UniswapV3Factory.spec.ts:132:7-132:44: `await factory.setOwner(other.address)` failed
$DIR/test/UniswapV3Factory.spec.ts:132:20-132:44: `.setOwner(other.address)` failed
$DIR/test/UniswapV3Factory.spec.ts:143:7-143:44: `await factory.setOwner(other.address)` failed
$DIR/test/UniswapV3Factory.spec.ts:143:20-143:44: `.setOwner(other.address)` failed
$DIR/test/UniswapV3Factory.spec.ts:144:27-144:52: `.setOwner(wallet.address)` failed
$DIR/test/UniswapV3Factory.spec.ts:153:27-153:56: `.enableFeeAmount(1000000, 10)` failed
$DIR/test/UniswapV3Factory.spec.ts:156:27-156:51: `.enableFeeAmount(500, 0)` failed
$DIR/test/UniswapV3Factory.spec.ts:159:27-159:55: `.enableFeeAmount(500, 16834)` failed
$DIR/test/UniswapV3Factory.spec.ts:162:7-162:44: `await factory.enableFeeAmount(100, 5)` failed
$DIR/test/UniswapV3Factory.spec.ts:162:20-162:44: `.enableFeeAmount(100, 5)` failed
$DIR/test/UniswapV3Factory.spec.ts:163:27-163:52: `.enableFeeAmount(100, 10)` failed
$DIR/test/UniswapV3Factory.spec.ts:166:7-166:44: `await factory.enableFeeAmount(100, 5)` failed
$DIR/test/UniswapV3Factory.spec.ts:166:20-166:44: `.enableFeeAmount(100, 5)` failed
$DIR/test/UniswapV3Factory.spec.ts:173:7-173:45: `await factory.enableFeeAmount(250, 15)` failed
$DIR/test/UniswapV3Factory.spec.ts:173:20-173:45: `.enableFeeAmount(250, 15)` failed
